.github/workflows/release.yml

```diff
@@ -51,368 +51,368 @@ jobs:
-    strategy:
-      matrix:
-        runtime: ['moonbase-alphanet']
-    steps:
-      -
-        uses: actions/checkout@v2
-      -
-        uses: actions/download-artifact@v2
-        with:
-          name: moonbase-alphanet
-          path: build/alphanet
-      -
-        name: Set up Ruby 2.7
-        uses: actions/setup-ruby@v1
-        with:
-          ruby-version: 2.7
-      -
-        name: Get runtime version
-        id: get-runtime-ver
-        run: |
-          runtime_ver="$(ruby -e 'require "./scripts/github/lib.rb"; puts get_runtime("${{ matrix.runtime }}")')"
-          echo "::set-output name=runtime_ver::$runtime_ver"
-      -
-        name: Upload ${{ matrix.runtime }} wasm
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ needs.publish-draft-release.outputs.asset_upload_url }}
-          asset_path: build/alphanet/${{ matrix.runtime }}-runtime.wasm
-          asset_name: ${{ matrix.runtime }}-runtime-v${{ steps.get-runtime-ver.outputs.runtime_ver }}.wasm
-          asset_content_type: application/wasm
-      -
-        name: Upload ${{ matrix.runtime }} node
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ needs.publish-draft-release.outputs.asset_upload_url }}
-          asset_path: build/alphanet/${{ matrix.runtime }}
-          asset_name: ${{ matrix.runtime }}
-          asset_content_type: application/octet-stream
-      -
-        name: Upload ${{ matrix.runtime }} specs plain
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ needs.publish-draft-release.outputs.asset_upload_url }}
-          asset_path: build/alphanet/${{ matrix.runtime }}-specs-plain.json
-          asset_name: ${{ matrix.runtime }}-specs-plain.json
-          asset_content_type: application/json
-      -
-        name: Upload ${{ matrix.runtime }} specs raw
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ needs.publish-draft-release.outputs.asset_upload_url }}
-          asset_path: build/alphanet/${{ matrix.runtime }}-specs-raw.json
-          asset_name: ${{ matrix.runtime }}-specs-raw.json
-          asset_content_type: application/json
-      -
-        name: Upload ${{ matrix.runtime }} genesis
-        uses: actions/upload-release-asset@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          upload_url: ${{ needs.publish-draft-release.outputs.asset_upload_url }}
-          asset_path: build/alphanet/${{ matrix.runtime }}-genesis.txt
-          asset_name: ${{ matrix.runtime }}-genesis.txt
-          asset_content_type: text/plain
+  # build:
```

Comment:
Why is it all commented ?

Reply:
It is just temporary so I'm not eating up cpu time running the build job every time I change whitespace. It will be removed.

Reply:
Your idea about `#no-build` in the commit message would be good here too.